contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -888,7 +887,6 @@ contract MinterSEAV0 is ReentrancyGuard, MinterBase, IFilteredMinterSEAV0 {
         );
         // load entire auction into memory
         auction = _projectConfig.activeAuction;
-        return auction;
```

Comment:
Nit: in this case would it be more consistent with our typical use of style to remove the named param in the return above instead and still return explicitly here?

Reply:
yeah, good point. I've reverted this to be unchanged, thanks!